File: solutions/c/easy/sum-numbers-c/solution-approach1.c
// ──────────────────────────────────────────────────
// Problem     Sum and Difference of Two Numbers
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    c
// Status      Accepted
// Submitted   2026-06-09, 12:59 a.m.
// ──────────────────────────────────────────────────

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
	 int a, b;
    float c, d;

    scanf("%d %d", &a, &b);
    scanf("%f %f", &c, &d);

    printf("%d %d\n", a + b, a - b);
    printf("%.1f %.1f\n", c + d, c - d);

    
    return 0;
}

File: solutions/python3/easy/python-print/solution-approach1.py
# ──────────────────────────────────────────────────
# Problem     Print Function
# Difficulty  Easy
# Subdomain   Introduction
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-06-21, 10:18 p.m.
# ──────────────────────────────────────────────────

if __name__ == '__main__':
    n = int(input())
      
    # Print numbers from 1 to n consecutively without spaces
    for i in range(1, n + 1):
        print(i, end="")
